lib/plugins/aws/package/index.js

```diff
@@ -102,6 +114,11 @@ class AwsPackage {
         await this.saveServiceState();
         return this.serverless.pluginManager.spawn('aws:common:moveArtifactsToPackage');
       },
+
+      'success': () => {
```

Comment:
I've added a `success` hook as `finalize` is already very widely used and I didn't want to create confusion. I'm not fully sold on the name though so I'm up for suggestions

Reply:
Yes `finalize` is used as token for many lifecycle event groups, but technically it's similar with `initialize` (it's maybe used less, as has use cases, but it's used)

Having I think it'll be consistent to follow _global_ `initialize` with _global_  `finalize` as it follows terminology that's used with hooks. `success` is something new and it's semantics doesn't necessary indicate it's about something to be invoked at the end.

Reply:
changed to `finalize` :+1: